packages/utils/src/lib/report.unit.test.ts

```diff
@@ -223,3 +229,37 @@ describe('getPluginNameFromSlug', () => {
     expect(getPluginNameFromSlug('plugin-b', plugins)).toBe('Plugin B');
   });
 });
+
+describe('sortAuditIssues', () => {
+  it('should sort issues by severity and source file', () => {
+    const mockIssues = [
+      { severity: 'warning', source: { file: 'b' } },
+      { severity: 'error', source: { file: 'c' } },
+      { severity: 'error', source: { file: 'a' } },
+      { severity: 'info', source: { file: 'b' } },
+    ] as Issue[];
+    const sortedIssues = [...mockIssues].sort(sortAuditIssues);
+    expect(sortedIssues).toEqual([
+      { severity: 'error', source: { file: 'a' } },
+      { severity: 'error', source: { file: 'c' } },
+      { severity: 'warning', source: { file: 'b' } },
+      { severity: 'info', source: { file: 'b' } },
+    ]);
+  });
+
+  it('should sort issues by source file and source start line', () => {
+    const mockIssues = [
+      { severity: 'info', source: { file: 'b', position: { startLine: 2 } } },
+      { severity: 'info', source: { file: 'c', position: { startLine: 1 } } },
+      { severity: 'info', source: { file: 'a', position: { startLine: 2 } } },
+      { severity: 'info', source: { file: 'b', position: { startLine: 1 } } },
+    ] as Issue[];
+    const sortedIssues = [...mockIssues].sort(sortAuditIssues);
+    expect(sortedIssues).toEqual([
+      { severity: 'info', source: { file: 'a', position: { startLine: 2 } } },
+      { severity: 'info', source: { file: 'b', position: { startLine: 1 } } },
+      { severity: 'info', source: { file: 'b', position: { startLine: 2 } } },
+      { severity: 'info', source: { file: 'c', position: { startLine: 1 } } },
+    ]);
+  });
+});
```

Comment:
Maybe good to also add a test case where there's a file but no position (it's optional after all), to document how we handle it (I assume missing start lines goes first).

Reply:
Isn't that tested in the first test? L234-L248.

Reply:
But there could be one where some of the issues have the `position` and some don't.

Reply:
Expanded the sorting conditions and tests to cover all cases.